test/e2e/serial/e2e_serial_test.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package serial
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	_ "github.com/openshift-kni/numaresources-operator/test/e2e/serial/tests"
```

Comment:
good to know this trick with "_"

Reply:
from https://go.dev/ref/spec#Import_declarations:
```
An import declaration declares a dependency relation between the importing and imported package. It is illegal for a package to import itself, directly or indirectly, or to directly import a package without referring to any of its exported identifiers. To import a package solely for its side-effects (initialization), use the [blank](https://go.dev/ref/spec#Blank_identifier) identifier as explicit package name:
```

this will trigger the `Describe` and create the specs to be used by gingko (it's more complex than this but that's roughly the idea)

Reply:
also: https://onsi.github.io/ginkgo/#adding-specs-to-a-suite

Reply:
thanks for the links